### PR TITLE
Add output viz tests for vtk data & test that the generated files can be read by paraview

### PR DESCRIPTION
**Pull request by jakebolewski**
Output is added as a summary of each run (similar to all the other CI assets).  We'll have to play around with the camera to make it reasonable for both 2D and 3D (or have a cli arg) but this should get us started.  Tests that the output files can be read by paraview and the pvpython script is useful for debugging.

https://github.com/CliMA/ClimaCore.jl/actions/runs/1665961301

Ex:
sphere vtu

### CODE REVIEW

**Comment by valeriabarra** [comment]
This is nice, but yes, as you mentioned, the camera for the 2D tests should point not in the YZ-plane as it currently is, otherwise if we also want to visually inspect the plots (other than checking if paraview can open them) you can't really see anything. Is it easy to set a default camera view?

**Comment by jakebolewski** [comment]
Yes I only looked at getting the sphere to work.  See the script

File: lib/ClimaCoreVTK/test/runtests.jl [hunk 1]
```diff
@@ -4,8 +4,14 @@ using IntervalSets
...
 
+running_CI() = !isempty(get(ENV, "CI", ""))
 
-dir = mktempdir()
+dir = if running_CI()
+    mkdir(joinpath(@__DIR__, "output"))
+else
+    mktempdir()
+end
+mkdir(joinpath(dir, "series"))
```
Comment (simonbyrne) [inline lib/ClimaCoreVTK/test/runtests.jl:14]: It is sometimes useful to save the results when running the test locally as well. Maybe we could do something like
```suggestion
dir = get(ENV, "CI_OUTPUT_DIR", mktempdir())
```
and then just set the environment variable in the .yml file.

Comment (jakebolewski) [reply lib/ClimaCoreVTK/test/runtests.jl:14]: Good suggestion, I'll also these for the other artifacts.

**Comment by jakebolewski** [comment]
changed the default view to XZ as it is the most useful.

**Pull request merged**